mirrulations-client/src/mirrclient/client.py

```diff
@@ -124,7 +190,9 @@ def send_job(self, job, job_result):
             data['directory'] = self.path_generator.get_path(job_result)
 
         self._put_results(data)
-        self.put_results_to_mongo(data)
+        # self.put_results_to_mongo(data)
+        # Instead we should increment the redis counter for things downloaded
+
         comment_has_attachment = self.does_comment_have_attachment(job_result)
```

Comment:
Everything from line 196 to 204 might be better placed in its own function maybe `put_results`